op-e2e/e2eutils/disputegame/output_game_helper.go

```diff
@@ -553,6 +559,36 @@ func (g *OutputGameHelper) PreimageExistsInOracle(ctx context.Context, data *typ
 	return exists
 }
 
+func (g *OutputGameHelper) UploadPreimage(ctx context.Context, data *types.PreimageOracleData, privateKey *ecdsa.PrivateKey) {
+	oracle := g.Oracle(ctx)
+	call := oracle.AddGlobalDataTxCall(data)
+	candidate, err := call.ToTxCandidate()
+	g.require.NoError(err, "Failed to create tx candidate")
+
+	logger := testlog.Logger(g.t, log.LvlInfo).New("role", "OutputGameHelper", "game", g.addr)
+	conf := txmgr.NewCLIConfig(g.system.NodeEndpoint("l1"), txmgr.DefaultChallengerFlagValues)
+	conf.NumConfirmations = 1
+	conf.ReceiptQueryInterval = time.Second * 1
+	conf.PrivateKey = e2eutils.EncodePrivKeyToString(privateKey)
+	txMgr, err := txmgr.NewSimpleTxManager("e2etest", logger, metrics.NoopMetrics, conf)
+	g.require.NoError(err, "Failed to create tx manager")
+
+	receipt, err := txMgr.Send(ctx, candidate)
+	g.require.NoError(err, "Failed to send tx")
+	g.require.EqualValues(gethtypes.ReceiptStatusSuccessful, receipt.Status, "Transaction failed")
+}
+
+func (g *OutputGameHelper) Oracle(ctx context.Context) *contracts.PreimageOracleContract {
```

Comment:
Ideally we wouldn't expose the oracle contract to tests and would just provide a high level function here that does what we need.